backend/mach.ml

```diff
@@ -67,6 +70,8 @@ type operation =
                 mode : Lambda.alloc_mode }
   | Iintop of integer_operation
   | Iintop_imm of integer_operation * int
+  | Iintop_atomic of { op : atomic_integer_operation;
```

Comment:
We can use `Cmm.atomic_integer_operation` here.

Reply:
Changed